jena-base/src/main/java/org/apache/jena/atlas/lib/BitsInt.java

```diff
@@ -155,14 +155,27 @@ boolean test(int bits, int value, int start, int finish)
      *  @param finish
      *  @return int
      */
-    
+
     public static final
     int access(int bits, int start, int finish)
     {
         check(start, finish) ;
-        return access$(bits, start, finish) ; 
+        return access$(bits, start, finish) ;
+    }
+
+    /**
+     * Clear the bit specified.
+     *  @param bits
+     *  @param bitIndex
+     *  @return int
+     */
+    public static final
+    int clear(int bits, int bitIndex)
+    {
+        check(bitIndex);
+        return clear$(bits, bitIndex) ;
```

Comment:
`clear$` already does the `check(bitIndex)`. Would it be better to have it only there, or is it better to be safe and perform that verification on both methods?

Reply:
Fixed.